service/collector/set.go

```diff
@@ -12,6 +12,8 @@ type SetConfig struct {
 	G8sClient versioned.Interface
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
+
+	WatchNamespace string
```

Comment:
Adding the watchNamesapce to follow up the `app` CR location.

Reply:
Why do we need that though? Why can't we list all apps from all namespaces?

Reply:
Can we do that using k8sclient?

Reply:
Okay I found at https://github.com/kubernetes/client-go/issues/159#issuecomment-288624475
I will follow it as all namespaces.

Reply:
Yeah but again, can you please explain why we have the namespace limitation at all in the first place? There might be a reason for that. So before we change code back and forth let'S understand why things are in a certain way.

Reply:
`app-operator` is watching the namespace (e.g. giantswarm) which is the user set as a flag. So I thought that `collector` also should do the same thing.

Reply:
I understand that, but that does not explain why it is like that and we both do apparently not understand why it is like that. So it would be really good if you could figure out why it is like that, so that we can then figure out how it should be for the collector. Do not just follow existing code. Let's understand and have a reason. Talk to people and see what you can get together. Your team should be able to help you in the first place.

Reply:
I don't think this is the thing that I should ask my team to find out the reason since I wrote most of the codes in this operator perhaps the question was not clear. 
Anyway, In the current situation, `app` CR is installed inside cluster namespace, so namespace limitation inside `collector` is wrong I believe.

Reply:
> I don't think this is the thing that I should ask my team to find out the reason since I wrote most of the codes in this operator

And yet neither of us seems to have a convincing explanation. I stop bothering you now. Do what you think is right.